app/src/main/java/com/example/easycloset/SplashScreenActivity.java

```diff
@@ -2,13 +2,36 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.os.Handler;
+
+import java.util.Timer;
+import java.util.TimerTask;
 
 public class SplashScreenActivity extends AppCompatActivity {
+    private final Timer timer = new Timer();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_splashscreen);
+
+
+        timer.schedule(new TimerTask() {
+            @Override
+            public void run() {
+                Intent intent = new Intent(SplashScreenActivity.this, LoginActivity.class);
+                startActivity(intent);
+                finish();
+            }
+        },3000);
+    }
+
+
+    @Override
+    protected void onPause() {
+        timer.cancel();
```

Comment:
onPause would be called in every case when activity is going to background, so the issues is if you pause here but then activity go to foreground again then onCreate will not be called, which means timer would not restart. 
take a look at onStart method, which would be called multiple times.

Reply:
fixed